plugins/modules/scaleway_database_backup.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python
```

Comment:
This does not need to be a copy of the module, but a symlink to the real module.

Reply:
Hmmmm, I think symlinks don't work, [the tests](https://dashboard.zuul.ansible.com/t/ansible/build/f2e8156e56ad4d46b490695d13d5c8dd/console) fail since I deleted the file and replaced it with a relative symlink.

Reply:
Resolved - My bad, the link was broken